src/Veldrid.SDL2/Sdl2.Window.cs

```diff
@@ -213,14 +228,22 @@ public enum SDL_FullscreenMode : uint
         Windowed = 0,
         Fullscreen = 0x00000001,
         FullScreenDesktop = (Fullscreen | 0x00001000),
+    }
+
+    public unsafe struct SDL_DisplayMode
+    {
+        public uint format;
+        public int w;
+        public int h;
+        public int refresh_rate;
+        public void* driverdata;
     }
 
-    public unsafe struct SDL_DisplayMode
+    public struct SDL_Rect

```

Comment:
@trolleyman  @mellinoe  I was going to use this new API but then I noticed that rather than using (https://github.com/mellinoe/veldrid/blob/bab9e25d808352fdcc36852ee7c7a1e059c3fe70/src/Veldrid.SDL2/Rectangle.cs) we have created a new struct (i.e. SDL_Rect). If there any reason why we are not using Rectangle.cs struct? If there is no reason would it be okay if I create a PR to use Rectangle rather than SDL_Rect and remove SDL_Rect ( if no one else is using it ).


P.S. Sorry for late comment (i.e. after PR is merged)

Reply:
Good catch, I fixed that and it uses Rectangle now.